Automattic/jetpack
# Codeclimate: rely on eslint 5 to avoid build errors

#### Changes proposed in this Pull Request:

This may help solve current errors when trying to run Codeclimate tests.

```
/usr/src/app/node_modules/eslint/lib/config/plugins.js:122
                throw err;
                ^

Error: Failed to load plugin lodash: Cannot find module 'eslint-plugin-lodash'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.require (module.js:468:17)
    at require (internal/module.js:20:19)
    at Object.module.exports.load (/usr/src/app/node_modules/eslint/lib/config/plugins.js:114:26)
    at Array.forEach (native)
    at Object.module.exports.loadAll (/usr/src/app/node_modules/eslint/lib/config/plugins.js:136:21)
    at Object.load (/usr/src/app/node_modules/eslint/lib/config/config-file.js:500:21)
    at loadConfig (/usr/src/app/node_modules/eslint/lib/config.js:64:33)
    at getLocalConfig (/usr/src/app/node_modules/eslint/lib/config.js:126:23)
```

CodeClimate's eslint 5 implementation supports `eslint-plugin-lodash`:
https://docs.codeclimate.com/docs/eslint

#### Testing instructions:

* For now, I believe the only way to test will be to merge the PR first, then check https://codeclimate.com/github/Automattic/jetpack/builds to see if the latest build errors like before.
* Once it's been merged, we will also be able to nstall https://github.com/codeclimate/codeclimate, and run `CODECLIMATE_DEBUG=1 CONTAINER_TIMEOUT_SECONDS=1800 codeclimate analyze`

#### Proposed changelog entry for your changes:

* None
